### DISCUSSION TITLE
HelloGitHub Badge

### DISCUSSION THREAD

**Discussion by 521xueweihan** [johnthagen/min-sized-rust#69] [General]
<img width="761" alt="image" src="https://github.com/user-attachments/assets/be5f8881-1adb-497c-b957-180d05c6c069">

🎉 Congratulations! Your [project](https://hellogithub.com/en/repository/483ad68b1a224afc93b1f3bf1e2b7e4c) has been featured and recommended by the HelloGitHub community. We invite you to join the HelloGitHub Badge Program. Joining will grant you the following privileges:

- **Community Recognition**: The badge indicates that your project has successfully passed the HelloGitHub community's stringent selection and recommendation process.
- **Increased Exposure**: Displaying the badge will draw more traffic to your project, attracting additional users and contributors.
- **Enhanced Interaction**: Users can quickly understand your project through the badge and engage with it (like, comment, bookmark).
- **Feedback Collection**: Gather genuine feedback from a broad user base, aiding in the continuous improvement of your project.
- **Special Identification**: Once verified, your comments will feature a distinctive mark and be prioritized for pinning.

📌 [Click here](https://hellogithub.com/en/repository/483ad68b1a224afc93b1f3bf1e2b7e4c/embed) to wear the badge and join the HelloGitHub Badge Program, allowing your open-source project to shine even brighter.

**Comment by johnthagen** [comment]
Thanks for the invitation. At this time, I'll decline.

**Reply by 521xueweihan** [reply]
@johnthagen Thank you for your reply.

If this message has caused you any distress, I sincerely apologize. However, this is not a mass marketing message but a choice I have made after careful consideration.

The [HelloGitHub](https://github.com/521xueweihan/HelloGitHub) open source project, which I have been continuously maintaining for 9 years, has allowed me to deeply understand the difficulties of building an open source project from scratch. Therefore, I hope to contribute to the min-sized-rust open source project, to help more people become aware of and fond of it, which is why I am willing to promote it.

At present, the HelloGitHub open source project has 80k stars, the website has a monthly PV of 400k, and there are 500k fans across the network. It is a pity that HelloGitHub has been spreading only in China before. Recently, I have just translated the website and content into English, preparing to let friends from abroad know what HelloGitHub has been doing.

**Reply by 521xueweihan** [reply]
Finally, it doesn't matter whether you wear the HelloGitHub badge or not. Enjoy the joy of open source instead of being affected by some bad things. Thank you for share min-sized-rust which helps many people.

**Reply by wcampbell0x2a** [reply]
> darktable open source project

🤔 🤔 🤔
